app/services/proofing/mock/resolution_mock_client.rb

```diff
@@ -27,7 +27,7 @@ class ResolutionMockClient < Proofing::Base
         elsif first_name.match?(/Time/i)
           raise Proofing::TimeoutError, 'resolution mock timeout'
 
-        elsif applicant[:ssn].match?(/6666/)
+        elsif !ssn.start_with?('900') && !IdentityConfig.store.test_ssn_allowed_list.include?(ssn)
```

Comment:
**suggestion:** it would be great to add a test for this branch (e.g. stub out `IdentityConfig.store` or something)

Reply:
good call, thank you.

added test in 4214fce